### PR TITLE
Un-cache CredentialManager if issuer re-issues

### PR DESCRIPTION
**Pull request by ianco**
Signed-off-by: Ian Costanzo <ian@anon-solutions.ca>

Fix for issue https://app.zenhub.com/workspaces/von---verifiable-organization-network-5adf53987ccbaa70597dbec0/issues/bcgov/theorgbook/880

### CODE REVIEW

File: server/vcr-server/agent_webhooks/views.py [hunk 1]
```diff
@@ -441,6 +441,10 @@ def handle_register_issuer(message):
     issuer_manager = IssuerManager()
     updated = issuer_manager.register_issuer(message)
 
+    # reset the global CredentialManager instance (to clear the CredentialType cache)
+    global credential_manager
```
Comment (nrempel) [inline server/vcr-server/agent_webhooks/views.py:445]: Would it make sense to not use global, and instead initialize the `CredentialManager` in each handler?

Comment (ianco) [reply server/vcr-server/agent_webhooks/views.py:445]: Does this lose the benefit of caching the credential types?

**Comment by ianco** [comment]
Upon further testing this is not working as planned, back to the drawing board!

**Comment by ianco** [comment]
Upon re-testing it's working fine!

**Pull request merged**